langchain/src/tools/index.ts

```diff
@@ -44,3 +44,4 @@ export {
 export { SearxngSearch } from "./searxng_search.js";
 export { SearchApi, type SearchApiParameters } from "./searchapi.js";
 export { formatToOpenAIFunction } from "./convert_to_openai.js";
+export { PythonInterpreterTool } from "./pyinterpreter.js";
```

Comment:
We can't export from here if it requires an optional peer dep, we'll need to use a custom entrypoint